module/hooks/json-changes.mjs

```diff
@@ -0,0 +1,102 @@
+import { DevMode } from '../classes/DevMode.mjs';
+
+Hooks.once('ready', () => {
+  // Default values to inject into arrays
+  const injectKeys = {
+    packs: {
+      private: false
+    },
+    dependencies: {
+      type: "module"
+    }
+  };
+
+  // Values to delete from arrays to compensate for diffObject problems
+  const cullKeys = {
+    //packs: ['private']
+  };
+
+  /**
+   * @param {String} file File to fetch
+   * @param {Object} current Currently loaded data.
+   * @param {Function} comparer Custom comparer instead of default diffObject
+   */
+  function fetchData(file, current, comparer) {
```

Comment:
declare `fetchData` outside the hook function, it's fine they're named generically as this this inside an esmodule.